### PR TITLE
PR for Cypress/fe 1508 decimal

### PR DESCRIPTION
**Pull request by DlgSHi**
Automation tests for Storybook components

Added to testing scope:

Decimal,
fixed locators for dialogFullScreen

### CODE REVIEW

File: cypress/integration/decimal.feature [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+  #     | ÄÖÜßäöüß                 |
+  #     | <>                       |
+
+  @positive
+  Scenario Outline: Change Decimal component label width
+    When I set label to "<label>"
+      And I set label help to "<label>"
+      And I check label inline checkbox
+      And I set label width slider to <width>
+    Then labelWidth on preview is <width>
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/integration/decimal.feature]: fixed

File: cypress/integration/decimal.feature [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+    Examples:
+      | label                    | width |
+      | Sample text              |  0    |
+      | 1234567890               |  10   |
+      | áéíóú¿¡üñ                |  50   |
+      | !@#$%^*()_+-=~[];:.,?{}  |  100  |
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/integration/decimal.feature]: agree with U, fixed

File: cypress/support/step-definitions/dialog-full-screen-steps.js [hunk 3]
```diff
@@ -1,13 +1,13 @@
-import { dialogFullScreenChildren, dialogFullScreenPreview } from '../../locators/dialog-full-screen';
+import { dialogFullScreenChildren, dialogFullScreenPreview, dialogFullScreenPreviewClosedState } from '../../locators/dialog-full-screen';
 
 Then('Dialog Full Screen children on preview is {string}', (children) => {
-  dialogFullScreenChildren().should('have.text', children);
+  dialogFullScreenChildren().children().should('have.text', children);
```
[the remark this answers is not in the document]
Comment (marcinstachowicz) [reply cypress/support/step-definitions/dialog-full-screen-steps.js]: when I removed `.children()` then scenario works correctly, coud you please re-check that?
second solution is locator change `cy.iFrame(DIALOG_FULL_SCREEN_CHILDREN).then($element => $element.children());`

[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/support/step-definitions/dialog-full-screen-steps.js]: fixed :)

File: cypress/integration/decimal.feature [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+      And I check label inline checkbox
+      And I set label align "<labelAlign>"
+    Then labelAlign on preview is set to "<labelAlign>"
+    Examples:
+      | label                    | labelAlign |
+      | áéíóú¿¡üñ                |  left      |
+      | 1234567890               |  right     |
+
+  @positive
+  Scenario Outline: Check Decimal component input field with different precision
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/integration/decimal.feature]: fixed, tests were added

**Pull request merged**